.github/workflows/daff.yml

```
@@ -0,0 +1,45 @@
+name: Comment Diff
+
+on:
+  pull_request_target:
+    paths:
+      - 'Spreadsheet/**'
+
+jobs:
+  comment-diff:
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        # The following allows for each run step to utilize ~/.bash_profile
+        # for setting up the per-step initial state.
+        # --login: a login shell. Source ~/.bash_profile
+        # -e: exit on first error
+        # -o pipefail: piped processes are important; fail if they fail
+        shell: bash --login -eo pipefail {0}
+    permissions:
+      pull-requests: write
+
+    steps:
+    - uses: actions/checkout@v3
```

GitHub has been sending around emails about other old versions of actions going away or being deprecated, so let's use the latest one here if we can.
```suggestion
    - uses: actions/checkout@v4
```